source/FingerBoard-Core/FBFlowInstance.class.st

```diff
@@ -207,7 +224,9 @@ FBFlowInstance >> printOn: aStream [
 ]
 
 { #category : #'as yet unclassified' }
-FBFlowInstance >> pushExecutor: anExecutor [ 
+FBFlowInstance >> pushExecutor: anExecutor [
+ 
+	executors add: anExecutor.
 	executor := anExecutor 
 		flowInstance: self;
 		suspendedExecutor: executor;
```

Comment:
why do we still need the suspendedExecutor. Isn't the list of executors not a replacment for that?

Reply:
There. is still one executor stepping at a time. When one executor go over all his steps, we still need a mechanism to jump back to the parent executor. This is done by restoring the #suspendedExecutor as current executor of the flow instance